values_test.go

```diff
@@ -72,3 +72,14 @@ func parseTime(s string) time.Time {
 	}
 	return t
 }
+
+// Indirect calls to findTestFile, used by TestFindTestFile. It is important
```

Comment:
NIT: Can you put these in a new file, maybe called `indirect_test.go`?

Reply:
Done.